### DISCUSSION TITLE
Custom name for installed libraries

### DISCUSSION THREAD

**Discussion by XanderC94** [microsoft/vcpkg#20678] [Q&A]
Looking among discussions and issue I haven't found a systematic or programmatically way to install libraries with a given name.

For example:

Leptonica, OpenCv and Tesseract are always installed with their version number trailing the library name, like tesserarct41.lib or tesseract41d.lib, leptonica-1.80.lib, etc... I would like to have uniform naming for the libraries in order, if needed, to roll updated versions of the libraries without changing my project configuration.

Another example is giflib that compiles both release and debug with the same name.

Sorry for the bother but I really couldn't find anything or how to better word what I needed!

**Comment by autoantwort** [comment]
No there is no setting to change that. You have to change the portfile.cmake of the ports to change the naming of the libs.

**Reply by XanderC94** [reply]
Ok, it will take some effort but better than nothing; thank you! 👍🏼
